nile-frontend/src/App.js

```diff
@@ -13,15 +13,15 @@ function App() {
                 <Header />
 
                 <Switch>
-                    <Route path="/singleProductListing">
+                    <Route exact path="/singleProductListing">
```

Comment:
i would perhaps make this something more precise like "/product"

Reply:
There is one with /listing, so Charles, Stephen and I chose this name